Server/main.cpp

```diff
@@ -121,8 +113,6 @@ void http_session(tcp::socket socket) {
 			res.prepare_payload();
 			http::write(socket, res);
 		}
-
-
 	}
 	catch (std::exception& e) 
```

Comment:
```suggestion
	catch (const std::exception& e) 
```

Ты же не собираешься его менять - тогда зачем получать изменяемую ссылку?

Некоторые вещи смогут пролезть в неизменяемую ссылку, а в изменяемую - не смогут